core/src/main/kotlin/ch/ergon/dope/resolvable/operator/FunctionOperator.kt

```diff
@@ -4,14 +4,7 @@ import ch.ergon.dope.DopeQuery
 import ch.ergon.dope.resolvable.expression.unaliased.aggregator.AggregateQuantifier
 
 interface FunctionOperator {
-    fun toFunctionQueryString(symbol: String, vararg arguments: DopeQuery, extra: DopeQuery?) =
-        if (extra == null) {
-            toFunctionQueryString(symbol = symbol, *arguments)
-        } else {
-            toFunctionQueryString(symbol = symbol, *arguments, extra)
-        }
-
-    fun toFunctionQueryString(symbol: String, vararg arguments: DopeQuery) = arguments.joinToString(
+    fun toFunctionQueryString(symbol: String, vararg arguments: DopeQuery?) = arguments.filterNotNull().joinToString(
```

Comment:
small thing, can you remove in the function below:
`fun toFunctionQueryString(symbol: String, quantifier: AggregateQuantifier?, vararg arguments: String) =`
the _vararg_ from the last argument? I think it is always called with one argument

Reply:
Yes I can 😄 and I did